Lecture-4/Task1/l4task1/Application.java

```diff
@@ -0,0 +1,44 @@
+package l4task1;
+
+import java.util.Scanner;
+
+public class Application {
+    private static Scanner scanner = new Scanner(System.in);
+    private static String input;
+
+    private static void showReverseSentence() {
+        StringBuffer stringBuffer = new StringBuffer(input);
```

Comment:
Use `StringBuilder` instead of `StringBuffer` when synchronization is not needed - as in this case